bin/regressive_test.sh

```diff
@@ -14,6 +14,17 @@ fi
 
 ./recompile.sh
 . ./install_dbtoaster.sh
+# check if scala installed
+if hash scala 2>/dev/null; then
+    version=$(scala -version 2>&1 | awk -F " " '{print $5}') 
+    if [[ $version != "2.10"* ]]; then
```

Comment:
Thanks for the reply @khuevu. Maybe for now we should keep it at > 2.10? I think the default Scala version in Ubuntu is 2.9, and I'm not sure if it would work then so it could be good to have the check.